microsite/sidebars.json

```diff
@@ -0,0 +1,17 @@
+{
+  "docs": {
```

Comment:
Will this be able to reference existing docs inside `../docs`?

Reply:
Yes. We tested this locally and it is able to pick up docs from docs folder. Needs a bit more work (like adding an id to each .md file for docusaurus to be able to pick it up.

Right now we are not exposing a top-level entry to docs yet. The aim right now is to make microsite work as is from the main repo and then add an entry point to docs.